service/webhook.go

```diff
@@ -0,0 +1,56 @@
+package service
+
+import (
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+)
+
+type GithubWebhookUserManagePayload struct {
+	Action string `json:"action"`
+	Scope  string `json:"scope"`
+
+	Member struct {
+		ID    uint   `json:"id"`
+		Login string `json:"login"`
+		Type  string `json:"type"`
+		Url   string `json:"html_url"`
+	}
+
+	Sender struct {
+		ID    uint   `json:"id"`
+		Login string `json:"login"`
+		Type  string `json:"type"`
+	}
+
+	Team struct {
+		ID   uint   `json:"id"`
+		Name string `json:"name"`
+		Slug string `json:"slug"`
+	}
+
+	Organization struct {
+		ID    uint   `json:"id"`
+		Login string `json:"login"`
+		URL   string `json:"url"`
+	}
+}
+
+func WebhookHandler(ctx *gin.Context, data *GithubWebhookUserManagePayload) {
```

Comment:
Here is just a question, do we need to verify errors here?

For example, like this

```go
func WebhookHandler(ctx *gin.Context, data *GithubWebhookUserManagePayload) error {
	switch data.Action {
	case "removed":
		if err := deleteUser(data); err != nil {
			return err
		}
	case "added":
		if err := addUser(data); err != nil {
			return err
		}
	default:
		return errors.New("unsupported action")
	}
	return nil
}
```

Reply:
> Here is just a question, do we need to verify errors here?
> 
> For example, like this
> 
> ```go
> func WebhookHandler(ctx *gin.Context, data *GithubWebhookUserManagePayload) error {
> 	switch data.Action {
> 	case "removed":
> 		if err := deleteUser(data); err != nil {
> 			return err
> 		}
> 	case "added":
> 		if err := addUser(data); err != nil {
> 			return err
> 		}
> 	default:
> 		return errors.New("unsupported action")
> 	}
> 	return nil
> }
> ```

Yes I agree but for now we don't have implementation of those methods so we can suppose that gonna be returned at least (error) and we can handle but probably to do it precisely we need to wait

Reply:
Perfect, makes sense! I am going to keep that in the radar so the following tasks I can mention that!